dev_guide/builds.adoc

```diff
@@ -366,13 +370,14 @@ trigger on:
 ====
 
 ----
-{
-  "metadata":{
-    "name": "ruby-20-centos7",
-  },
   "kind": "ImageStream",
-  "apiVersion": "v1beta1",
-}
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "ruby-20-centos7"
+  },
+  "status": {
+    "dockerImageRepository": ""
+  }
```

Comment:
this example is broken...  (it was broken even before you touched it).  this ImageStream doesn't actually point anywhere (no docker repository).  looks like it was supposed to point to ruby-20-centos7.